uzupelnianka.txt

```diff
@@ -1,20 +1,22 @@
-Klasa to …
+Klasa to podstawowy element programowania objektowego, który może zawierać zmienne i metody slużące
+do operowania na tych zmiennych
 
-Obiekt to …
+Obiekt to element stworzony przez konstruktor klasy. Objekt daje dostep do metod zdefiniowanych w klasie
```

Comment:
Najważniejesze, żeby zapamiętać, że obiekt to jedna instancja, a klasa to szablon